Makefile.core.mk

```diff
@@ -29,6 +29,7 @@ testgrid:
 	@GOARCH=amd64 GOOS=linux go run k8s.io/test-infra/testgrid/cmd/configurator --prow-config prow/config.yaml --prow-job-config prow/cluster/jobs --output-yaml --yaml testgrid/default.yaml --oneshot --output testgrid/istio.gen.yaml
 
 generate-config:
+	@rm -fr prow/cluster/jobs/istio/*/*.gen.yaml
```

Comment:
Can you put this file in a subfolder so it doesn't require root level approval?

Reply:
There's already a line in CODEOWNERS for that.